File: Swift-C++-IR-Comparison/test/structsAndClasses/Swift/Classes/OverrideConvenienceInit.swift
@main
struct MyMain {

    static func main() {
        let burger = Food(name: "Tasty")
        let random = Food()
    }
}

class Food {
    var name: String
    init(name: String) {
        self.name = name
    }
    convenience init() {
        self.init(name: "[Unnamed]")
    }
}

File: AutoTranspiledTests/try-0/OverrideConvenienceInit.cpp

#include <iostream>
#include <string>

using namespace std;

class Food {
    public:
        string name;
        Food(string name) {
            this->name = name;
        }
        Food() {
            this->name = "[Unnamed]";
        }
};

int main() {
    Food burger = Food("Tasty");
    Food random = Food();
    
    return 0;
}